src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -3,33 +3,75 @@ import React, { FC } from "react"
 import styled from "styled-components"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Details_conversation } from "v2/__generated__/Details_conversation.graphql"
+import { Media } from "v2/Utils/Responsive"
 
-const BorderedFlex = styled(Flex)`
+export const DETAIL_BOX_ANIMATION = `transition: width 0.3s ;`
+const DETAIL_BOX_XS_ANIMATION = `transition: opacity 0.3s;`
+
+const DetailsContainer = styled(Flex)<{ opacity?: 0 | 1 }>`
   border-left: 1px solid ${color("black10")};
   flex-shrink: 0;
-  margin-left: -1px;
+  ${DETAIL_BOX_ANIMATION}
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  background-color: ${color("white100")};
+`
+// XS screen has opacity animation instead of width
+const XSDetailsContainer = styled(DetailsContainer)`
+  ${DETAIL_BOX_XS_ANIMATION}
+  opacity: ${({ opacity }) => opacity};
 `
```

Comment:
For these, could you potentially use a single container but switch between the necessary styles using `${media.xs``}` (similar to [this](https://github.com/artsy/force/blob/3dce74fee97984bbc0e507ee5d3d2ce781a74cc9/src/v2/Apps/Conversation/Components/Reply.tsx#L16-L19)) instead of having two separate containers and relying on `Media`? Generally if we can avoid `Media`, I believe that we should (being that it double renders).